vspace/src/main/webapp/WEB-INF/views/exhibition/aboutPagePublic.html

```diff
@@ -0,0 +1,49 @@
+
+<html layout:decorate="~{layouts/main}">
+<head>
+</head>
+
+<body>
+
+<div layout:fragment="content" class="main-content">
+<script th:inline="javascript">
+var aboutPageDescription = [[${aboutPageText}]];
+function openNav(){
+	$("#mySidenav").css("width", "250px");
+}
+function closeNav(){
+	$("#mySidenav").css("width", "0");
+}
+
+
+
+ $(document).ready(function() {
+	 var divElement = document.getElementById("aboutPageDescription"); 
+	 divElement.innerHTML += aboutPageDescription;
+}); 
+</script>
```

Comment:
why are you setting the content via javascript instead of directly in the html?

Reply:
the aboutPageDescription will have the content like this, <p>hello<strong>How are you</strong>doing</p> as it is taken from the markdown editor. So i thought it is better to set it using javascript and append it to the inner html. Should I change this?

Reply:
I don't really see a reason to use javascript here unless there is an issue with adding the html into the template html.

Reply:
Removed it